packages/util/src/elements.ts

```
@@ -0,0 +1,508 @@
+      CORRECT: 0,
+      total: 0,
+    }
+  } else if (
+    (elementData.type === PrismaElementType.SC ||
+      elementData.type === PrismaElementType.MC ||
+      elementData.type === PrismaElementType.KPRIM) &&
+    'choices' in elementData.options
+  ) {
+    const choices = elementData.options.choices.reduce(
+      (acc: Record<string, number>, choice: Choice) => ({
+        ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)